test/e2e/specs/dashboard/settings__site-visibility.ts

```diff
@@ -82,12 +83,20 @@ describe( 'Dashboard: Site Visibility Settings', function () {
 		} );
 	} );
 
+	beforeEach( async function () {
+		incognitoContext = await browser.newContext();
```

Comment:
I think we can move this to `beforeAll`, since we're only opening a page from this context and there’s no login action involved 🤔